src/pg/test/e2e.test.js

```diff
@@ -1049,4 +1049,100 @@ describe('pg_e2e', () => {
       );
     });
   });
+  describe('jsonpath operators', () => {
```

Comment:
This isn't JsonPath, that is something else. We can call it json key existence operators, or just json key operators.

It does not support multi level paths, only one level keys.